cpp/include/raft/handle.hpp

```diff
@@ -32,10 +32,10 @@
 ///@todo: enable once we have migrated cuml-comms layer too
 //#include <common/cuml_comms_int.hpp>
 
-#include <raft/linalg/cublas_wrappers.h>
-#include <raft/linalg/cusolver_wrappers.h>
 #include <raft/sparse/cusparse_wrappers.h>
 #include <raft/comms/comms.hpp>
+#include <raft/linalg/cublas_wrappers.hpp>
```

Comment:
These should be in detail. They aren't intended to be public and are only meant to be used as a facade for the actual cublas/cusolver/cusparse/etc... calls.

Reply:
Oh yep, you are totally right - missed that one my bad

Reply:
Jus to update this conversation so it's not lost over the holidays, the intention here is to wrap the standard cuda library functions so we centralize all the calls but to provide RAFT public API functions which are completely agnostic of the standard cuda libraries underneath.

As discussed in https://github.com/rapidsai/cuml/pull/4446, for a short time, we might have a few places in the cuml solver code which is calling down to raft's `linalg::detail` namespace. These will need to be updated to use the public API, once it's available (and we do that in a follow-on PR).